src/coll/mod.rs

```diff
@@ -1029,6 +1033,7 @@ impl Collection {
             cmd,
             CommandType::ListIndexes,
             self.read_preference.to_owned(),
+            Some(1)
```

Comment:
Just pass `None` here; the idea behind having the function in the cursor module accept an `Option` as a parameter is to avoid having to specify the default elsewhere.